iFixit/react-commerce
# Use Algolia secured API key

fixes #257 

## Changelog

- Refactored logic to account for Algolia secured key generation
- Refactored env variables
- Added github secret for `ALGOLIA_API_KEY`
- Added Vercel env variable for `ALGOLIA_API_KEY`

## QA

1. Visit the Vercel PR preview (replace vercel.app with cominor.com)
2. Visit a few product list pages and verify visible product parts reflect the current product list
